src/test/java/com/global/api/tests/realex/RealexCreditTests.java

```diff
@@ -212,6 +212,71 @@ public void creditAuthorization_WithMultiAutoSettle() throws ApiException {
         assertNotNull(response);
         assertEquals("00", response.getResponseCode());
     }
+    
+    @Test
+    public void creditAuthorization_WithMultiAutoSettle_WithEstNumTxn() throws ApiException {
+        Transaction response = card.authorize(new BigDecimal("14"))
+                .withEstimatedTransactions(3)
+                .withCurrency("USD")
+                .withMultiCapture(true)
+                .execute();
+        assertNotNull(response);
+        assertEquals("00", response.getResponseCode());
+    }
+    
+    @Test
+    public void creditMultisettle() throws ApiException {
+        Transaction response = card.authorize(new BigDecimal("17"))
+                .withEstimatedTransactions(3)
+                .withCurrency("USD")
+                .withMultiCapture(true)
+                .execute();
+        assertNotNull(response);
+        assertEquals("00", response.getResponseCode());
+
+        Transaction rebate = response.multicapture(new BigDecimal("17"))
```

Comment:
I'm not sure about having this as a separate method as I'd prefer this be an extension of the existing `capture` method that is already exposed. @russelleverett thoughts?

Reply:
I initially started that way, but realized that single capture does not need the authcode, nor the amount, but multisettle do.
It would then become possible to build a multisettle without those parameters as the capture method would allow it. Having a dedicated method requiring the mandatory parameters would prevent that.
I am not sure what is you take on letting the host respond with errors vs forcing the dev to use the "right" method...